torch/jit.py

```diff
@@ -1,26 +1,496 @@
 import torch.autograd.function as function
 import torch._C
 from torch.autograd import Variable
-from torch.nn import Module
+from torch.nn import Module, ParameterList, Parameter
+from torch._six import raise_from
 from collections import defaultdict
 import itertools
 import types
 import contextlib
 import os
-import torch.contrib._graph_vis as graph_vis
-# Example how to use:
+import functools
+import inspect
+
+
+class Placeholder(object):
+    def __init__(self, s):
+        self.s = s
+
+    def __str__(self):
+        return self.s
+
+    def __repr__(self):
+        return self.s
+
+
+HOLE = Placeholder("HOLE")
+VOLATILE = Placeholder("VOLATILE")
+
+
+# TODO: verify is not implemented yet
+def compile(arg=None, verify=False, **kwargs):
```

Comment:
Yes. It's so that you can both say `@torch.jit.compile` as well as `@torch.jit.compile(nderivs=2)`; people always forget the parens in the no-argument case.